docs/sources/features/datasources/x-ray.md

```diff
@@ -0,0 +1,139 @@
+| Credentials profile name | Specify the name of the profile to use (if you use `~/.aws/credentials` file), leave blank for default.                  |
+| Assume Role Arn          | Specify the ARN of the role to assume.                                                                                    |
+| External ID              | If you are assuming a role in another account, that has been created with an external ID, specify the external ID here. |
+
+## Authentication
+
+In this section we will go through the different type of authentication you can use for X-Ray data source.
+
+### AWS credentials
+
+If the Auth Provider is `Credentials file`, then Grafana tries to get credentials in the following order:
+
+- Environment variables (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`)
+- Hard-code credentials
+- Shared credentials file
+- IAM role for Amazon EC2
+
+Refer to [Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html) in the AWS documentation for more information.
+
+### AWS credentials file
+
+Create a file at `~/.aws/credentials`. That is the `HOME` path for user running grafana-server.
+
+> **Note:** If the credentials file in the right place, but it is not working, then try moving your .aws file to '/usr/share/grafana/'. Make sure your credentials file has at most 0644 permissions.
+
+Example credential file:
+
+```bash
+[default]
+aws_access_key_id = <your access key>
+aws_secret_access_key = <your access key>
+region = us-west-2
+```
+
+## Query editor 
+
+The most important field in the editor is the query type. There are four query types:
+- Trace List (Traces in AWS)
+- Trace Statistics
+- Trace Analytics (Analytics in AWS)
+- Insights
+
+{{< docs-imagebox img="/img/docs/v72/x-ray-query-editor.png" max-width="800px" class="docs-image--left" caption="X-Ray query editor" >}}
+
+### Trace List
+
+The Trace List is pretty much the same as in AWS. Clicking on the trace id in the first column opens the trace on the right side. Notice the query field in the editor which works the same as in AWS. You can write queries, filter expressions, or a trace ID there.
+
+{{< docs-imagebox img="/img/docs/v72/x-ray-trace-list.png" caption="X-Ray Trace List in Grafana" >}}
+
+### Trace Statistics
+
+In Trace Statistics you can see a graph and a table showing information about error, fault, throttle, success, and total count. You can use the columns field in the query editor to only see specified columns.
+
+{{< docs-imagebox img="/img/docs/v72/x-ray-trace-statistics.png" caption="X-Ray Trace Statistics in Grafana" >}}
+
+### Trace Analytics
+
+In Trace Analytics you can visualize one of the tables from Analytics in AWS.
+
+### Insights
+
+In Insights you can see the summary table for Insights just like in AWS. Clicking the InsightId will take you to AWS console.
```

Comment:
> just like in AWS

Again this can be removed. We can probably link to x-ray insights documentation once that is ready.

TODO: we should check if we should actually keep this or what is the state of insights in x-ray